### ISSUE TITLE
Shotgun Casings are not correct/present in game files and do not make a sound when dropping on ground

### ISSUE THREAD

**Issue by PENGUANO** [yvt/openspades#909]
The casings in the shotgun's files are just weird "X" shapes, and are 3x3x1, and are not actual casings like you see in the SMG and Rifle's files. They also do not make any sound, or come out of the shotgun when shot.

**Comment by yvt** [comment]
The "X" model is used as a placeholder for GPL-incompatible assets. It's supposed to be "overridden" by `pak000-Nonfree.pak` with real ones and never to be seen by end users. How did you install OpenSpades?

**Comment by PENGUANO** [comment]
How did you install OpenSpades?

Using the website on windows, if that's what you're getting at. Keep in mind I didn't go to school for coding so I may be lack luster here, but I do know a bit

**Comment by PENGUANO** [comment]
> The "X" model is used as a placeholder for GPL-incompatible assets. It's supposed to be "overridden" by `pak000-Nonfree.pak` with real ones and never to be seen by end users. 

Are you saying that shotguns are not supposed to eject casings/shells in the first place?

**Comment by yvt** [comment]
> > The "X" model is used as a placeholder for GPL-incompatible assets. It's supposed to be "overridden" by `pak000-Nonfree.pak` with real ones and never to be seen by end users.
> 
> Are you saying that shotguns are not supposed to eject casings/shells in the first place?

No, what I meant is the "X" model is not supposed to be seen when OpenSpades is installed in an officially recommended way. 

And actually, yes, shotguns aren't supposed to eject shells in the first place because we haven't implemented that.

**Comment by PENGUANO** [comment]
> > > The "X" model is used as a placeholder for GPL-incompatible assets. It's supposed to be "overridden" by `pak000-Nonfree.pak` with real ones and never to be seen by end users.
> 
> > 
> 
> > Are you saying that shotguns are not supposed to eject casings/shells in the first place?
> 
> 
> 
> No, what I meant is the "X" model is not supposed to be seen when OpenSpades is installed in an officially recommended way. 
> 
> 
> 
> And actually, yes, shotguns aren't supposed to eject shells in the first place because we haven't implemented that.

oh ._. Ok, so then I have two  main questions:
1. Is the X an issue or something I should be concerned about
2. Will Shotgun casings/shells be added in a future update?
And I also have a minor question:
A. If there is nothing about adding the shotgun casing on this GitHub, should I add that? I have models I made already for this exact thing, if you would like them
![Uploading 93736118...]
(Idk if what I put above loaded correctly, it was supposed to be a screenshot of the shell I made, but I'm doing this on mobile so it may not send).
